source/documentation/obtain-a-bearer-token.html.md.erb

```diff
@@ -0,0 +1,30 @@
+---
+title: Obtain a bearer token
+weight: 20
+---
+
+# Obtain a bearer token
```

Comment:
Not fully read the steps below as I assume we are throwing the Authority Wizard away

Reply:
Replaced with Create a Test User